news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,133 @@
+
+- Error page in the Back office: If an unexpected error occurs in the back office when running in production mode, you are now presented with a useful error page instead of a blank page, this way you have more information to understand what is happening.
+
+- It is now possible to translate email contents in the Back Office, when using the new Email theme feature introduced in 1.7.6 and available in Design > Email theme 
+
+- A lot of [international improvements](https://build.prestashop.com/news/prestashop-keeps-growing-internationally/) have been made. New languages have been added to PrestaShop and its installer, localization packs have been updated to keep PrestaShop up-to-date to offer accurate international data for all businesses.
+
+
+### Improved core modules
+
+Some modules have been improved with bug fixes for more robustness and new features for more productivity :
+
+- Theme customization: Now available for product & category pages (in addition to the home page), you can easily configure and customize your product and categories’ pages theme and main native modules.
+
+- Block reassurance: Now available on all pages (above or below the header and / or the footer) in addition to product pages and during the checkout. You can customize the color of the icon, add a description and a link. With this module you build the confidence of your visitors and thus improve your conversion rate.
+
+
+### Notable fixes
+
+More than [160 bugs](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aclosed+label%3Abug+milestone%3A1.7.7.0+-label%3ARegression+) have been fixed for the 1.7.7 (vs 110 for the 1.7.6.0), including [10 highly expected bug fixes](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aclosed+label%3ATopwatchers+label%3Afixed+milestone%3A1.7.7.0), [5 front-office notable fixes](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AFO+label%3Amust-have+label%3Afixed+milestone%3A1.7.7.0+-label%3Atopwatchers+) and [5 back-office notable fixes](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABO+label%3Amust-have+label%3Afixed+milestone%3A1.7.7.0+-label%3Atopwatchers+-label%3Aregression)
+
+
+### Notable technical improvements
+
+- **Updated PHP support.** Following [our announcement last year](https://build.prestashop.com/news/announcing-end-of-support-for-obsolete-php-versions/), PrestaShop 1.7.7 introduces support for PHP  7.3 while dropping support for PHP versions earlier than 7.1.3.
+
+- **Support for emojis.** All database tables are now encoded in utf8bm4 by default, enabling support for emojis everywhere 
+
+- **Updated jQuery to 3.4.1 everywhere.** Thanks to [jquery migrate](https://github.com/jquery/jquery-migrate), we have been able to upgrade jQuery to the latest version in Back Office as well as Classic & Core theme, without introducing backwards incompatible changes.
```

Comment:
from what @jolelievre told me, we are not 100% sure that there won't be any incompatibility issues.